<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QPushButton" name="DecBtn">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="CloseBtn">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="IncBtn">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="BgridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="Blue">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="BspinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSlider" name="BverticalSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="GREENgridLayout">
        <item row="1" column="0">
         <widget class="QSpinBox" name="GspinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="Glabel">
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSlider" name="GverticalSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="Rlabel">
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="RspinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="RverticalSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QSpinBox" name="AmntSpinbox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Increase or decrease by: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>249</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BverticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BspinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GverticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GspinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BspinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BverticalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GspinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GverticalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RspinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RverticalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RverticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RspinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
